sunbeam_migrate/tests/integration/handlers/octavia/test_load_balancer.py

```diff
@@ -0,0 +1,421 @@
+        interval=2,
+        wait=CONF.load_balancer_migration_timeout,
+    )
+    return session.load_balancer.get_pool(pool.id)
+
+
+def _create_test_member(session, lb, pool_id, address):
+    subnet_id = lb.vip_subnet_id
+    member = session.load_balancer.create_member(
+        pool_id,
+        name=test_utils.get_test_resource_name(),
+        subnet_id=subnet_id,
+        address=address,
+        protocol_port=80,
+    )
+    session.load_balancer.wait_for_load_balancer(
+        lb.id,
+        status="ACTIVE",
+        failures=["ERROR"],
+        interval=2,
+        wait=CONF.load_balancer_migration_timeout,
+    )
+    return session.load_balancer.get_member(member.id, pool_id)
+
+
+def _create_test_health_monitor(session, lb_id, pool_id):
+    health_monitor = session.load_balancer.create_health_monitor(
+        name=test_utils.get_test_resource_name(),
+        pool_id=pool_id,
+        type="TCP",
+        delay=5,
+        timeout=5,
+        max_retries=3,
+    )
+
+    session.load_balancer.wait_for_load_balancer(
+        lb_id,
+        status="ACTIVE",
+        failures=["ERROR"],
+        interval=2,
+        wait=CONF.load_balancer_migration_timeout,
+    )
+    return session.load_balancer.get_health_monitor(health_monitor.id)
+
+
+def _check_migrated_load_balancer(source_lb, destination_lb):
+    fields = [
+        "name",
+        "description",
+        "is_admin_state_up",
+        "vip_address",
+    ]
+    for field in fields:
+        source_attr = getattr(source_lb, field, None)
+        dest_attr = getattr(destination_lb, field, None)
+        assert source_attr == dest_attr, f"{field} attribute mismatch"
+
+
+def _cleanup_destination_load_balancer(session, lb_id):
+    """Best-effort removal of listeners, pools, members, then the LB itself."""
+    lb = session.load_balancer.get_load_balancer(lb_id)
+    if not lb:
+        return
+
+    session.load_balancer.wait_for_load_balancer(
+        lb_id,
+        status="ACTIVE",
+        failures=["ERROR"],
+        interval=2,
+        wait=CONF.load_balancer_migration_timeout,
+    )
+
+    for listener in session.load_balancer.listeners(loadbalancer_id=lb_id):
+        pool_id = getattr(listener, "default_pool_id", None)
+        if pool_id:
+            pool = session.load_balancer.get_pool(pool_id)
+            if pool:
+                for member in session.load_balancer.members(pool.id):
+                    try:
+                        session.load_balancer.delete_member(
+                            member.id, pool.id, ignore_missing=True
+                        )
+                        session.load_balancer.wait_for_load_balancer(
+                            lb_id,
+                            status="ACTIVE",
+                            failures=["ERROR"],
+                            interval=2,
+                            wait=CONF.load_balancer_migration_timeout,
+                        )
+                    except Exception:
```

Comment:
Silently ignoring "NotFound" errors when performing cleanup is ok, but not any exception.

We can "pass" for NotFound exception and log a warning for other exception types, telling the user that some resources may have been leaked.

The same applies to other similar try/except blocks.